Dockerfile

```diff
@@ -1,18 +1,19 @@
-FROM node:10-alpine3.10
+FROM node:10
```

Comment:
Curious. Is there any reason you are no longer using the `alpine` version?
Also, I couldn't find it in the Referee docs, but is there any reason we are using node10? It's going to be deprecated soon: https://nodejs.org/en/about/releases/

Reply:
Mostly that I didn't have to add dependencies, I should probably be explicit, this uses Debian stretch. As per the node version, that is a good point, I'll check it works as intended and update it.